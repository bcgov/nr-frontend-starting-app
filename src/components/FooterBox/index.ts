import styled from 'styled-components';

const FooterBox = styled.footer`
  bottom: 0;
  position: fixed;
  width: 100%;
  background-color: #036;
  border-top: 2px solid #fcba19;
  color: #fff;
  font-family: "Noto Sans", Verdana, Arial, sans-serif;
  height: 46px;
  display: flex;
  align-items: center;
`;

export default FooterBox;
