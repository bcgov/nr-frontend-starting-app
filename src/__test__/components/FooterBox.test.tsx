/* eslint-disable no-undef */
import React from 'react';
import renderer from 'react-test-renderer';
import FooterBox from '../../components/FooterBox';

it('renders correctly', () => {
  const tree = renderer
    .create(<FooterBox />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});
