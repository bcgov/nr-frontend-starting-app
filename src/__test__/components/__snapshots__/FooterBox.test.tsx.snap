// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the FooterBox component should match the snapshot 1`] = `
<footer
  className="sc-bczRLJ bpbShr"
/>
`;
